reactive/coredns.py

```diff
@@ -25,16 +60,24 @@ def start_charm():
                     'username': image_info.username,
                     'password': image_info.password,
                 },
-                'command': ['/coredns', '-dns.port={}'.format(port)],
+                'args': ["-conf", "/etc/coredns/CoreFile"],
                 'ports': [
+                    {'name': 'dns', 'containerPort': int(hookenv.config('port')), 'protocol': 'UDP'},
+                    {'name': 'dns-tcp', 'containerPort': int(hookenv.config('port')), 'protocol': 'TCP'},
+                    {'name': 'metrics', 'containerPort': 9153, 'protocol': 'TCP'},
+                ],
+                'readinessProbe': {
+                    'httpGet': {'path': '/ready', 'port': 8181, 'scheme': 'HTTP'},
+                },
```

Comment:
Should there be a livenessProbe too? I see one in the upstream [coredns template](https://github.com/coredns/deployment/blob/8fb8871a309cc77baaef27f5b227ec0e546daf0c/kubernetes/coredns.yaml.sed#L133).